test/functional/test-extension-location.js

```diff
@@ -40,7 +40,7 @@ describes.sandboxed('Extension Location', {}, () => {
         pathname: 'examples/ads.amp.html',
         host: 'localhost:8000',
         protocol: 'http:',
-      }, 'amp-ad', true);
+      }, 'amp-ad', '0.1', true);
```

Comment:
Added test for '1.0', but since we made `extensionVer` mandatory in `calculateExtensionScriptUrl`, I can't add one where the version is undefined.